### DISCUSSION TITLE
Weird appearance of a two digit number before first bibliography item in final references

### DISCUSSION THREAD

**Discussion by hildebrand-institute** [Zettlr/Zettlr#2594] [Q&A]
I guess this could be most probably related to pandoc than to Zettlr, but as with Zettlr 2.0 pandoc happens to be integrated within Zettlr, I report the issue. Well, using an "article" LaTeX model, as usual bibliographic references are listed at the end of the paper. With Zettlr 2.0 I get a weird two digit number (10) in front of the first bibliography item, as you can see from the attached pic. It is not related with the specific item or with the specific bib file. I cannot get rid of this stupid number! Within the default file I use: 
```
filters: 
- type: citeproc
```
![two_digit_number](https://user-images.githubusercontent.com/57742994/136687584-656680f5-0f47-4d5d-816d-4aba94540c8c.png)
Thanks

**Comment by nathanlesage** [comment]
Since it's only on one item, it seems to be some wrong field inside the citation index. How does that entry look in the references pane of Zettlr itself?

**Reply by hildebrand-institute** [reply]
Yes, that was of course my first bet, but it appears in fact in front of any other item alternatively being the first in the list and also using a totally different bib file. Furthermore, if no bibliographic item is present, even so I get a "00" under the "References" title...

**Reply by hildebrand-institute** [reply]
Wait a minute... No bibliographic item is shown within the Zettlr references pane (I've never taken a look at it before). And: 1) biblio entries are listed at the end of the paper; 2) but if I write the biblio entry in the body of the markdown file like e.g.: `[@author99, 10]`, then it is not rendered as a full biblio entry within a footnote, but only like: `(author_name, 10)` (even if within the LaTeX model I use biblatex with the `style=philosophy-verbose` option enabled... With previous Zettlr versions I've never had such weird results. Anyway, as soon as possible I will test everything also on Debian.

**Reply by nathanlesage** [reply]
Wait, are you using a BibLaTeX library, or a BibTeX library? There's actually a difference. Zettlr doesn't directly parse those files and uses the Astrocite-library for that. And Astrocite cannot really reliably read BibLaTeX files. Buuut that wouldn't explain why in your output you're seeing this weird result, since Pandoc works perfectly fine with BibLaTeX.

If the number 10 always appears, and if you get a 00 without anything, this almost looks like a rogue list enumeration o.O

**Reply by hildebrand-institute** [reply]
Yes, I use biblatex... So, it is normal I cannot see biblio entries in Zettlr panel, but all the other strange results are to be considered inexplicable at a first sight, right? Now, unfortunately, I'm at work in office and not at my desk philosophizing... As soon as possible I will test everything also with previous Zettlr versions... Many thanks

**Reply by nathanlesage** [reply]
Do so, and maybe even just check what happens if you switch from BibLaTeX to BibTeX. Maybe there's something going on either way? Feel free to check this!
